boost/teamcity_boost.cpp

```diff
@@ -71,7 +94,7 @@ struct TeamcityFormatterRegistrar {
     }
 };
 
-BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar)
+BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar);
 
 // Formatter implementation
 static std::string toString(boost::unit_test::const_string bstr) {
```

Comment:
Just wanted to keep the changes to a minimum for a single patch/feature.  IMO, things like moving functions around should be done in a separate patch, when possible.